From 847bbe6306abf09b4e8bb06c7a1053f5d5ea7dfc Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Sun, 11 Mar 2018 20:36:50 +0100
Subject: Do not remove read permission on cups-brf

cups checks only for executability and writability, while checksumming tools
need to be able to read files easily.

Closes: #878867
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 4ce78a6e8..8e03294fa 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1090,7 +1090,7 @@ endif
 	$(LN_S) -f pdf.utf-8.simple \
 		$(DESTDIR)$(pkgcharsetdir)/pdf.utf-8
 if ENABLE_BRAILLE
-	chmod g-rwx,o-rwx $(DESTDIR)/$(pkgbackenddir)/cups-brf
+	chmod g-wx,o-wx $(DESTDIR)/$(pkgbackenddir)/cups-brf
 endif
 
 
